src/secrets/CommandEnv.ts

```diff
@@ -283,7 +283,11 @@ class CommandEnv extends CommandPolykey {
                 // Formatting as a .env file
                 let data = '';
                 for (const [key, value] of Object.entries(envp)) {
-                  data += `${key}='${value}'\n`;
+                  if (options.envExport) {
+                    data += `export ${key}='${value}'\n`;
+                  } else {
+                    data += `${key}='${value}'\n`;
+                  }
```

Comment:
The if statement applies to Unix systems. On Windows, an equivalent check is unnecessary because the export keyword (or equivalent) isn’t needed. On Windows, child processes automatically inherit the parent process's environment variables, as documented on the official Microsoft website: [Environment Variables](https://learn.microsoft.com/en-us/windows/win32/procthread/environment-variables).

Reply:
If that is the case, then instead of making this an option, we can probably make this another env format - something like `--env-format unix-export`. Or we can also rename this to be `--env-format unix-local` for non-export and `--env-format unix-global` for exported environment variables. Thoughts, @CMCDragonkai?